tests/conftest.py

```diff
@@ -224,3 +226,15 @@ def reset_rat_cache_fx(rat_cache_fx: RatCache):
     yield
     # and clean up after ourselves
     rat_cache_fx.flush()
+
+
+@pytest.fixture(scope="session")
+def dbm_fx():
+    """returns a DBM instance"""
+    return DatabaseManager()
+
+
+@pytest.fixture(scope="session")
```

Comment:
Is there any reason for these fixtures to have the scope of session? This makes tests interdependent, as, if one test breaks the fixture object, it will be broken for all following tests as well. This should be avoided if at all possible.

Reply:
The scope isn't supposed to be set there...

Reply:
It actually has a reason.
1. It is used by a also session scoped fixture (setUp/tearDown of the DBM tests)
1. It's hard to break the DBM
1. This makes the tests faster as it re-uses the same instance for all tests